app/views/admin_outgoing_message/edit.html.erb

```diff
@@ -42,10 +50,25 @@
 </p>
 
 <%= form_tag admin_outgoing_message_path(@outgoing_message), :method => 'delete' do %>
-  <div>
-    <%= submit_tag "Destroy outgoing message",
-          :class => "btn btn-danger",
-          :data => { :confirm => "This is permanent! Are you sure?" } %>
+  <div class="control-group">
+    <div class="controls">
+      <% if @is_initial_message %>
+        <%= submit_tag "Destroy outgoing message",
+                       :class => "btn btn-danger",
+                       :disabled => true,
+                       :data => {
+                         :confirm => "This is permanent! Are you sure?" } %>
+        <span class="help-inline">
+          The initial outgoing message cannot be destroyed. You can set the
+          prominence to <tt>hidden</tt> instead.
```

Comment:
This is going to be confusing when the "initial" request [is a followup](https://gist.github.com/garethrees/b38b5cfa8236c3ca1f84aecf09d20602). Maybe it might be better to say "_This_ outgoing message cannot be destroyed"? Thoughts welcome here.